gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -43,6 +43,14 @@ export interface IFocusableControl {
     * @returns an array of controls
     */
     keepsFocusWith(): Nullable<Control[]>;
+    /**
+    * Function to focus the control programmatically
+    */
+    focus(): void;
```

Comment:
not sure to understand why we need it at ADT level?

Reply:
The `IFocusableControl` interface is defined & exported here since adt uses it a bit - do you want to migrate it to a separate file?

Reply:
If you don't mind :)

Reply:
Moved interface to `gui/src/2D/controls/focusableControl.ts`